app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java

```diff
@@ -55,6 +60,7 @@ public void setUp() {
         serviceWrite = new Retrofit.Builder()
             .baseUrl(APIUtils.GET_API)
```

Comment:
Should we use the production API for testing? Shouldn't we use the testing URL i.e. `https://world.openfoodfacts.net` host? Current implementation leads to the creation of real product in our prod db. https://world.openfoodfacts.org/product/0978020137962/coca3-auchan. This product is created by the `post_product()` test.